alembic/versions/384567ab231a_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""initial migration
+
+Revision ID: 384567ab231a
+Revises: 
+Create Date: 2024-10-27 16:34:17.722794
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '384567ab231a'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is currently empty. You need to add the SQLAlchemy operations here to define how to revert the changes made in the `upgrade` function.